app/src/main/java/com/zeapo/pwdstore/ClipboardService.kt

```diff
@@ -92,7 +93,7 @@ class ClipboardService : Service() {
                 ClipboardUtils.clearClipboard(clipboardManager, deepClear)
             }
         } else {
-            Timber.tag("ClipboardService").d("Cannot get clipboard manager service")
+            tag("ClipboardService").d { "Cannot get clipboard manager service" }
```

Comment:
Timber-kt should detect the tag automatically in Debug mode. If this is all we want, we could remove the `tag(...)` here.